zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerMetrics.java

```diff
@@ -67,7 +67,10 @@
     SNAP_COUNT(new SimpleCounter("snap_count")),
     COMMIT_COUNT(new SimpleCounter("commit_count")),
     CONNECTION_REQUEST_COUNT(new SimpleCounter("connection_request_count")),
-    BYTES_RECEIVED_COUNT(new SimpleCounter("bytes_received_count"));
+    BYTES_RECEIVED_COUNT(new SimpleCounter("bytes_received_count")),
+
+    RESPONSE_PACKET_CACHE_HITS(new SimpleCounter("response_packet_cache_hits")),
+    RESPONSE_PACKET_CACHE_MISSING(new SimpleCounter("response_packet_cache_misses"));
```

Comment:
how well this cache works under different type of workloads, based on the stats here? (if it's ok to share)

Reply:
I don't have numbers to share at the moment. :(